File: app/src/main/java/in/woloo/www/dailylogscreen/models/DailyLogSymptoms.java
package in.woloo.www.dailylogscreen.models;

import java.util.Objects;

public class DailyLogSymptoms {
    public String subTitleName;

    public int imageURL;

    public String getSubTitleName() {
        return subTitleName;
    }

    public void setSubTitleName(String subTitleName) {
        this.subTitleName = subTitleName;
    }

    public int getImageURL() {
        return imageURL;
    }

    public void setImageURL(int imageURL) {
        this.imageURL = imageURL;
    }

    @Override
    public boolean equals(Object o) {
        if (this == o) return true;
        if (o == null || getClass() != o.getClass()) return false;
        DailyLogSymptoms that = (DailyLogSymptoms) o;
        return imageURL == that.imageURL && subTitleName.equals(that.subTitleName);
    }

    @Override
    public int hashCode() {
        return Objects.hash(subTitleName, imageURL);
    }
}

File: app/src/main/java/in/woloo/www/more/dailylogscreen/models/DailyLogSymptoms.kt
package `in`.woloo.www.more.dailylogscreen.models

import java.util.Objects

class DailyLogSymptoms {
    var subTitleName: String? = null

    var imageURL: Int = 0

    override fun equals(o: Any?): Boolean {
        if (this === o) return true
        if (o == null || javaClass != o.javaClass) return false
        val that = o as DailyLogSymptoms
        return imageURL == that.imageURL && subTitleName == that.subTitleName
    }

    override fun hashCode(): Int {
        return Objects.hash(subTitleName, imageURL)
    }
}
